inst/tutorials/bednets/bednets.Rmd

```diff
@@ -1,75 +1,211 @@
+simulation_output |>
+  mutate(total_mosquito_population = Sm_gamb_count + Im_gamb_count + Pm_gamb_count) |>
+  ggplot(aes(x = timestep, y = total_mosquito_population)) +
+  geom_line(linewidth = 1.2, colour = "deepskyblue2") +
+  theme_bw() +
+  labs(x = "Time (days)", y = "Adult Female Mosquito Population Size") +
+  scale_y_continuous(limits = c(0, 7500)) +
+  geom_vline(xintercept = c(365, 1460), linetype = "dashed")
+
+# Plot the mosquito population through time:
+simulation_output |>
+  mutate(pfpr210 = n_detect_lm_730_3650 / n_730_3650) |>
+  ggplot(aes(x = timestep, y = pfpr210)) +
+  geom_line(linewidth = 1.2, colour = "brown2") +
+  theme_bw() +
+  labs(x = "Time (days)", y = "Prevalence (2-10)") +
+  scale_y_continuous(limits = c(0, 1), expand = c(0, 0)) +
+  scale_x_continuous(expand = c(0, 0))
 
-```{r add-function, exercise=TRUE, exercise.lines = 5}
-add <- function() {
-  
-}
 ```
 
-## Topic 2
+Looking at the plots, we can see that the net distribution events (black dashed lines) cause a rapid decline in the adult female mosquito population size which results in a decline in prevalence. We can also see that the prevalence begins to rebound between the distribution events. Think about why this is occurring. What parameter(s) might affect the rate of this recovery? Using the code chunk above, try re-running the simulation with a different net `retention` time and see how it affects what we see.
+
+## Specifying a campaign when simulating multiple mosquito species
+
+The default behaviour is for `malariasimulation` to simulate a single mosquito species, *Anopheles gambiae*. However, as we've discussed, `malariasimulation` supports the simulation of settings with multiple mosquito species, and has in-built parameter sets for *An. funestus* (`fun_params`), *An. arabiensis* (`arab_params`), and *An. stephensi* (`steph_params`). In this section, we'll see how to set up the same bed net campaign we've just used, but with two vector species.
 
-### Exercise with Hint
+We won't focus on the parameters in these sets today (although feel free to ask later), but we can view the parameter sets:
 
-*Here's an exercise where the chunk is pre-evaluated via the `exercise.eval` option (so the user can see the default output we'd like them to customize). We also add a "hint" to the correct solution via the chunk immediate below labeled `print-limit-hint`.*
+```{r set_species_demo_1, exercise = TRUE}
 
-Modify the following code to limit the number of rows printed to 5:
+# View the parameter set for An. funestus:
+fun_params
+
+# ... and view the associated helper files:
+?fun_params
 
-```{r print-limit, exercise=TRUE, exercise.eval=TRUE}
-mtcars
 ```
 
-```{r print-limit-hint}
-head(mtcars)
+Similar to `set_drugs()` in the treatment exercise, the `set_species()` function is used to load the mosquito species parameters of interest into our list of `malariasimulation` parameters. The `set_species()` function has three arguments:
```

Comment:
Is it possible that attendess may have jumped here before seeing `set_drugs()`
```suggestion
The `set_species()` function is used to load the mosquito species parameters of interest into our list of `malariasimulation` parameters. The `set_species()` function has three arguments:
```

Reply:
Yeah quite possible - have made the change as suggested!